### PR TITLE
Fix contribution transformation

### PR DESCRIPTION
**Pull request by maipet**
See #1952.

### CODE REVIEW

**Comment by TobiasNx** [comment]
+1

**Comment by dr0i** [comment]
Will be deployed next Monday.

File: src/test/resources/alma-fix/99371463467006441.json [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  "@context" : "http://lobid.org/resources/context.jsonld",
+  "almaMmsId" : "99371463467006441",
+  "title" : "Training for group experience",
+  "alternativeTitle" : [ "Training for group experience" ],
+  "otherTitleInformation" : [ "A syllabus of materials from a laboratory course for group leaders given at Columbia University in 1927" ],
+  "publication" : [ {
+    "startDate" : "1929",
+    "type" : [ "PublicationEvent" ],
+    "location" : [ "New York :" ],
+    "publishedBy" : [ "Inquiry" ]
```
Comment (dr0i) [inline src/test/resources/alma-fix/99371463467006441.json:11]: Strange - this build succesfully with GitHub actions first - but after merging it fails because it expects `[ "New York" ]` (i.e. without colon).

Comment (TobiasNx) [reply src/test/resources/alma-fix/99371463467006441.json:11]: Okay it seems that the commit was not okay, I did no good review here.

For an better approach we should add the new test file in a separate first commit, if there is none yet.
In the later commit we should add the code changes and include the changes in the test to highlight the reaction.
